test/es2015.test.js

```diff
@@ -17,6 +17,7 @@ describe('when applied with uglifyOptions.ecma', () => {
     });
 
     new UglifyJsPlugin({
+      parallel: { cache: false, workers: 0 },
```

Comment:
Same comment I made for `empty-options.test`> It's important that these options for these tests are not changed in order for us to **really** understand that it is not breaking those use cases.